contrib/freshmaker-cli

```diff
@@ -225,6 +248,49 @@ def get_freshmaker_events_by_ids(api_url, ids):
     return events
 
 
+def get_advisories_from_errata(errata_url, date_from=date.today() - timedelta(7),
+                               date_to=date.today()):
+    """
+    Sends the GET request to the Errata filter for the latest RHSA, in SHIPPED_LIVE state. Returns
+    the list of advisories that are IMPORTANT or CRITICAL and the issue date is between specified
+    dates.
+    """
+    if (date_to - date_from).days > 7:
+        print("The interval between dates should not exceed 7 days")
+        sys.exit(1)
+
+    krb_auth = HTTPKerberosAuth()
+    r = requests.get(errata_url, auth=krb_auth, timeout=30)
+    r.raise_for_status()
+    data = r.json()
+    return [
```

Comment:
With the current amount of advisories shipped by ET it seems unlikely that FM will perform more than one request to get the latest advisories (within the maximum set to one week). But in the future this amount might increase... We could consider at least checking if we got all the results and see if a new request would be necessary. @mprahl wdyt? Should we still check the second page or is it so unlikely that this will happen that we shouldn't bother?

Reply:
Wanted to add. As we have in the filter 100 advisories, the oldest one is 1.5 month old. Will we need some older advisories ?

Reply:
I don't think we'll need older advisories. But my concern is that with time the number of advisories filed in ET could increase and maybe we'll need pagination. That's why I'd be interested in hearing @mprahl on this. If we think it's unlikely that ET will speed up so fast in years to come, then we can just leave it like this.

Reply:
Can the filter be set to be ordered by the timestamp of when it shipped? Then you can keep going to the next page until you encounter an advisory that was shipped outside of your date range.

Reply:
This filters can be set to be ordered by Release date or Last modified. Or The Newest/Oldest of course (now they are ordered by Newest )